manifest.webmanifest

```diff
@@ -0,0 +1,63 @@
+{
+  "lang": "en",
+  "dir": "ltr",
+  "name": "A-Painter",
+  "short_name": "A-Painter",
+  "description": "Paint in VR in your browser. Powered by A-Frame.",
+  "start_url": "./?src=manifest",
+  "display": "fullscreen",
+  "theme_color": "#0a0a0a",
+  "background_color": "#0a0a0a",
+  "icons": [
+    {
+      "src": "/img/icon-512.png",
```

Comment:
let me know if you have a better icon to use (possibly the `A` wordmark from the A-Frame splash screen)